x/ecocredit/server/core/features/msg_remove_class_creator.feature

```diff
@@ -0,0 +1,104 @@
+Feature: Msg/RemoveClassCreator
+
+  A class creator can be remove:
+  - when the class creator is exist
+  - when the authority is a governance account address
+  - the class creator is removed
+
+  Rule: The class creator is exist
+
+    Scenario: The class creator is exist
+      Given class creators with properties
+      """
+      {
+        "creators":[
+            "regen156d26rl52y3wl865pr5x9q2vqetuw9kf0642sa"
+        ]
+      }
+      """
+      When alice attempts to remove class creators with properties
+      """
+      {
+        "authority":"regen1nzh226hxrsvf4k69sa8v0nfuzx5vgwkczk8j68",
+        "creators":[
+            "regen156d26rl52y3wl865pr5x9q2vqetuw9kf0642sa"
+        ]
+      }
+      """
+      Then expect no error
```

Comment:
same here

Reply:
Same as above. This is correct but we should have a rule for `the class creator is removed` and check state there.

Reply:
Same as above. This is correct but we should have a rule for `the class creator is removed` and check state there. See other server feature files for reference. Each item listed should correspond with a rule.